tools/lib/glbToI3dm.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var Cesium = require('cesium');
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = glbToI3dm;
+
+/**
+ * Generates a new Buffer representing a i3dm asset.
+ *
+ * @param {Buffer} glbBuffer A buffer containing a binary glTF asset.
+ * @param {Buffer} [featureTableJSONBuffer] A buffer containing the batch table to use for the b3dm asset.
+ * @param {Buffer} [featureTableBinaryBuffer] A buffer containing the accompanying binary batch table section for the b3dm asset.
+ * @param {Buffer} [batchTableJSONBuffer] A buffer containing the batch table to use for the b3dm asset.
+ * @param {Buffer} [batchTableBinaryBuffer] A buffer containing the accompanying binary batch table section for the b3dm asset.
+ * @param {Number} [gltfFormat] The gltf format for the body of the glTF field.
+ * @returns {Buffer} Buffer representing the b3dm asset.
+ */
+function glbToI3dm(glbBuffer, featureTableJSONBuffer, featureTableBinaryBuffer, batchTableJSONBuffer, batchTableBinaryBuffer, gltfFormat) {
+    if (!defined(glbBuffer)) {
+        throw new DeveloperError('glbBuffer is not defined.');
+    }
+
+    batchTableJSONBuffer = defaultValue(batchTableJSONBuffer, new Buffer(0));
+    batchTableBinaryBuffer = defaultValue(batchTableBinaryBuffer, new Buffer(0));
+    featureTableJSONBuffer = defaultValue(featureTableJSONBuffer, new Buffer(0));
```

Comment:
By extension, we are throwing an error for [`gltfByteLength === 0`](https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/blob/master/tools/lib/extractI3dm.js#L50-L53).

So unless `glbToI3dm` is directly called by the user, bypassing `3d-tiles-tools.js`, the error would be thrown before glbToI3dm is called by `3d-tiles-tools.js`. Thus it would never actually execute the fallback to set  `POSITION` and `[0, 0, 0]` as you suggested.